UIMentoringSession/MenteesChallenges/Start Screen UI Challenge/Bilyk/MaksymBilyk_StartScreenViewController.xib

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Check image background content mode for small devices
<img width="1288" alt="Screenshot 2023-05-27 at 09 23 31" src="https://github.com/alexnikol/UIMentoringSessions/assets/14887101/2d82c7f5-673d-45b1-96cf-4a7a6d845c3a">